docs/software-installation-guide.md

```diff
@@ -3,45 +3,89 @@
 ## User installation
 
 (Optional but recommended) install [anaconda](https://www.anaconda.com/products/distribution) and create a virtual environment  
-```
+
+```sh
 conda create -y -n recOrder python=3.9
 conda activate recOrder
 ```
+
 Install `napari` and `recOrder-napari`:
-```
+
+```sh
 pip install "napari[all]" recOrder-napari
 ```
+
 Open `napari` with `recOrder-napari`:
-```
+
+```sh
 napari -w recOrder-napari
 ```
+
 View command-line help by running
-```
+
+```sh
 recOrder.help
 ```
 
-## Developer installation:
+## Developer installation
 
-Install [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and `conda` (either [anaconda](https://www.anaconda.com/products/distribution) or [miniconda](https://docs.conda.io/en/latest/miniconda.html)).
+### Windows (x86_64)
 
-Create a conda environment dedicated to `recOrder`:
-```
-conda create -y -n recOrder python=3.9
-conda activate recOrder
-```
+1. Install [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and `conda` (either [anaconda](https://www.anaconda.com/products/distribution) or [miniconda](https://docs.conda.io/en/latest/miniconda.html)).
 
-Clone this repository:
-```
-git clone https://github.com/mehta-lab/recOrder.git
-```
+2. Create a conda environment dedicated to `recOrder`:
 
-Install `recOrder` and its developer dependencies:
-```
-cd recOrder
-pip install -e ".[dev]"
-```
+    ```sh
+    conda create -y -n recOrder python=3.9
+    conda activate recOrder
+    ```
+
+3. Clone this repository:
+
+    ```sh
+    git clone https://github.com/mehta-lab/recOrder.git
+    ```
 
-To acquire data via `Micromanager`, follow  [microscope installation guide](./microscope-installation-guide.md).
+4. Install `recOrder` and its developer dependencies:
+
+    ```sh
+    cd recOrder
+    pip install -e ".[dev]"
+    ```
+
+5. To acquire data via `Micro-Manager`, follow  [microscope installation guide](./microscope-installation-guide.md).
 
 **Optional GPU**: `recOrder` supports NVIDIA GPU computation with the `cupy` package. Follow [these instructions](https://github.com/cupy/cupy) to install `cupy` and check its installation with ```import cupy```. To enable gpu processing, set ```use_gpu: True``` in the config files.
 
+### Mac OS X on Apple Silicon (arm_64)
+
+[Napari](napari.org) uses PyQt5 to draw its GUI, which is not currently compatible with `pip`. We have tested the following emulated development environment on Mac OS 12.5. (It is available natively  through [conda-forge](https://github.com/conda-forge/miniforge) if the user does not intend to edit the `recOrder-napari` source code.)
```

Comment:
- "PyQt5...is not currently compatible with `pip`" --- is this true? I think my current `recOrder` environment is running a `pip install`ed PyQt5?
- "It is available natively" - does "It" refer to PyQt5 here?
- extra space between natively and through

Reply:
I meant PyQt5 is not available natively with `pip`. You are right that the expression is not very clear here.